.github/workflows/release.yml

```diff
@@ -0,0 +1,16 @@
+name: Release to Marketplace
+on:
+  push:
+    tags: ["*"]
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: 11
+          cache: 'gradle'
+      - run: echo "RELEASE_VERSION=$(./scripts/version-from-git-tag.sh)-alpha" >> $GITHUB_ENV
```

Comment:
I see what `scripts/version-from-git-tag.sh` does.
What about the tags? The still need to be pushed manually (via `scripts/push-git-tag-for-next-release.sh`, not by the CI), right?

Reply:
Correct. You have to manually push the git tag, which requires running validation tests locally first, and then CI runs the build+publish.

Reply:
I will update contributing.md to reflect the new flow once I have validated everything works.